rclcpp/src/rclcpp/utilities.cpp

```diff
@@ -44,9 +44,37 @@
 #define HAS_SIGACTION
 #endif
 
+rclcpp::Logger g_logger = rclcpp::get_logger("rclcpp");
```

Comment:
@wjwwood consider making this a `static` member of the `SignalHandler` or exposing it as a singleton like in [line 81](https://github.com/ros2/rclcpp/pull/607/files#diff-fe7ed4c8f373fc8cba1b5a0c60312137R81).